_data/mentors.yml

```diff
@@ -194,4 +194,13 @@
   github: hossainalikram
   image: hossain.jpg
   twitter: Hossainalikram
+  bio: 
+  
+  - name: Doris Lee
```

Comment:
@mananwason @jig08 No need to revert the PR. This is not a syntax error right? This could be fixed in another commit.